packages/console/src/pages/UserDetails/index.tsx

```diff
@@ -70,7 +70,7 @@ const UserDetails = () => {
 
   const {
     field: { onChange, value },
-  } = useController({ name: 'customData', control, rules: { required: true } });
+  } = useController({ name: 'customData', control });
```

Comment:
wondering if it would be more convenient to allow the user to set this field empty.

Reply:
No, we provide a default value `{}` for this field.
We don't allow this field to be empty.

Reply:
Now we fall back on empty custom data to `{}`.